mod.ts

```diff
@@ -11,28 +12,39 @@ interface Parser {
 
 type LoadOptions = {
   configPath?: string;
+  env?: string;
```

Comment:
I didn't see it being used anywhere, did I miss something?

Reply:
For force setting ENV, instead of reading `DENO_ENV` env var, not implemented yet

Reply:
grate, so let's pass it for now. ..